lib/samples/show_extruded_features/show_extruded_features.dart

```diff
@@ -0,0 +1,171 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/common/common.dart';
+import 'package:flutter/material.dart';
+
+class ShowExtrudedFeatures extends StatefulWidget {
+  const ShowExtrudedFeatures({super.key});
+
+  @override
+  State<ShowExtrudedFeatures> createState() => _ShowExtrudedFeaturesState();
+}
+
+class _ShowExtrudedFeaturesState extends State<ShowExtrudedFeatures>
+    with SampleStateSupport {
+  // Create a controller for the scene view.
+  final _sceneViewController = ArcGISSceneView.createController();
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+  // An enum representing types of population statistics.
+  FilterType _filterType = FilterType.totalPopulation;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        left: false,
+        right: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  child: ArcGISSceneView(
+                    controllerProvider: () => _sceneViewController,
+                    onSceneViewReady: onSceneViewReady,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: FilterType.values
+                      .map<Widget>(
+                        (filterType) => ElevatedButton(
+                          onPressed: () =>
+                              changeExtrusionExpression(filterType),
+                          child: Text(filterType.name),
+                        ),
+                      )
+                      .toList(),
+                ),
+              ],
+            ),
```

Comment:
Please add a comment to explain what is happening here for someone looking to learn from the sample